statsmodels/tsa2/sarimax/params.py

```diff
@@ -0,0 +1,402 @@
+                       title='seasonal AR coefficients')
+
+        ar_stationary = True
+        seasonal_ar_stationary = True
+        if self.k_ar_params > 0:
+            ar_stationary = is_invertible(self.ar_poly.coef)
+        if self.k_seasonal_ar_params > 0:
+            seasonal_ar_stationary = is_invertible(self.seasonal_ar_poly.coef)
+
+        return ar_stationary and seasonal_ar_stationary
+
+    @property
+    def is_invertible(self):
+        """(bool) Is the reduced moving average lag poylnomial invertible."""
+        # Short-circuit if there is no MA component
+        validate_basic(self.ma_params, self.k_ma_params,
+                       title='MA coefficients')
+        validate_basic(self.seasonal_ma_params, self.k_seasonal_ma_params,
+                       title='seasonal MA coefficients')
+
+        ma_stationary = True
+        seasonal_ma_stationary = True
+        if self.k_ma_params > 0:
+            ma_stationary = is_invertible(self.ma_poly.coef)
+        if self.k_seasonal_ma_params > 0:
+            seasonal_ma_stationary = is_invertible(self.seasonal_ma_poly.coef)
+
+        return ma_stationary and seasonal_ma_stationary
+
+    def to_dict(self):
+        """
+        Return the parameters split by type into a dictionary.
+
+        Returns
+        -------
+        split_params : dict
+            Dictionary with keys 'exog_params', 'ar_params', 'ma_params',
+            'seasonal_ar_params', 'seasonal_ma_params', and (unless
+            `concentrate_scale=True`) 'sigma2'. Values are the parameters
+            associated with the key, based on the `params` argument.
+        """
+        return self._params_split.copy()
+
+    def to_pandas(self):
+        """
+        Return the parameters as a Pandas series.
+
+        Returns
+        -------
+        series : pd.Series
+            Pandas series with index set to the parameter names.
+        """
+        return pd.Series(self.params, index=self.param_names)
+
+    def __repr__(self):
+        """Represent SARIMAXParams object as a string."""
+        components = []
+        if self.k_exog_params:
+            components.append('exog=%s' % str(self.exog_params))
+        if self.k_ar_params:
+            components.append('ar=%s' % str(self.ar_params))
+        if self.k_ma_params:
+            components.append('ma=%s' % str(self.ma_params))
+        if self.k_seasonal_ar_params:
+            components.append('seasonal_ar=%s' %
+                              str(self.seasonal_ar_params))
+        if self.k_seasonal_ma_params:
+            components.append('seasonal_ma=%s' %
+                              str(self.seasonal_ma_params))
+        if not self.spec.concentrate_scale:
+            components.append('sigma2=%s' % self.sigma2)
+        return 'SARIMAXParams(%s)' % ', '.join(components)
+
+    def __str__(self):
+        """Represent parameter array as a string."""
+        return str(self.params)
```

Comment:
why not just alias `__repr__`?

Reply:
I think that aliasing `__repr__` is probably a better idea here, thanks.